netbox/utilities/templates/form_helpers/render_fieldset.html

```diff
@@ -0,0 +1,65 @@
+{% load i18n %}
+{% load form_helpers %}
+<div class="field-group mb-5">
+  {% if heading %}
+    <div class="row">
+      <h5 class="col-9 offset-3">{{ heading }}</h5>
+    </div>
+  {% endif %}
+  {% for layout, title, items in rows %}
+
+    {% if layout == 'field' %}
+      {# Single form field #}
+      {% render_field items.0 %}
+
+    {% elif layout == 'attribute' %}
+      {# A static attribute of the form's instance #}
+      <div class="row mb-3">
+        <label class="col-sm-3 col-form-label text-lg-end required">{{ title }}</label>
+        <div class="col">
+          <div class="form-control-plaintext">
+            {{ items.0|linkify }}
+          </div>
+        </div>
+      </div>
+
+    {% elif layout == 'inline' %}
+      {# Multiple form fields on the same line #}
+      <div class="row mb-3">
+        <label class="col col-3 col-form-label text-lg-end">{{ title|default:'' }}</label>
+        {% for field in items %}
```

Comment:
Maybe overkill, but this seems set for 3 wide, might be more flexible if could specify maybe 2 wide for some.  But could be future FR

Reply:
We want to ensure that all row labels render with the same column width to effect a nice vertical alignment.